op-service/eth/types.go

```diff
@@ -145,6 +145,30 @@ func (b Bytes8) TerminalString() string {
 	return fmt.Sprintf("0x%x", b[:])
 }
 
+type Bytes9 [9]byte
```

Comment:
Nothing to change here yet, but I wonder if we can get away with `[]byte` for the eip1559Params instead of `[8]byte` or [9]byte`.